### ISSUE TITLE
UI / functional testing of editable submission

### ISSUE THREAD

**Issue by jseraf** [NUARIG/competitions#365]
Related to #248 

Feedback for editable submissions branch. Workflow, displays, submit button actions, flash messages, etc.

Test from Grant admin and applicant side on a Test grant on staging / alpha. Application should have at least one required file upload question.

Under different Grant statuses, states, etc. (e.g. Open / Closed for submissions, Draft / Published)

Applicant: apply to grant, save as draft with/without required fields, save as draft replace file uploads, submit.  

Admin: display, distinction between states, etc.

**Comment by firaswehbe** [comment]
Depends on #248

**Comment by firaswehbe** [comment]
@mattbaumann1 Found first bug, please see: #373 

I'll assign to you.

**Comment by lmkw** [comment]
i can save my draft with upload now, but when i hit submit nothing seems to happen. opening another ticket as a bug #375

**Comment by mattbaumann1** [comment]
I am working on this in Ticket #367.

**Comment by mattbaumann1** [comment]
The deployed changes should fix the update issues mentioned above.

**Comment by firaswehbe** [comment]
I wonder if the best route to send the user to after saving a draft is to *stay* in the submission rather than go back to "My Submissions"

**Comment by firaswehbe** [comment]
Reopened because there is a PR that will be underway. Need to re-test after that is merged. ref: #367
